doc/architecture.mmd

```diff
@@ -0,0 +1,64 @@
+flowchart LR
+    U("<img src='https://super.so/icon/dark/user.svg'; width='25'/>User")
+    subgraph Node1
+        FS1[(FileSystem)]
+        P1[Pod]
+        D1[Daemon]
+        R1[Recorder]
+        D1 -.->|install profile|FS1
+        P1 -.-> |use profile|FS1
+        D1 --> |gRPC: record profile| R1
+        R1 --> |record profile| P1
+        D1 -->|get/watch| P1
+    end
+    subgraph Node2
+        FS2[(FileSystem)]
+        P2[Pod]
+        D2[Daemon]
+        R2[Recorder]
```

Comment:
If we make it that detailed that we want to add the recorder, then we have to add the others as well:

- Enricher
- Metrics
- Reconciler for Seccomp, SELinux and AppArmor

Reply:
should the name be fixed in architecture.md?